src/ruby/spec/generic/rpc_server_spec.rb

```diff
@@ -187,6 +187,23 @@ def server_sends_bad_input(_, _)
       end
       expect(&blk).to raise_error
     end
+
+    it 'can be created with arguments directly passed from environment' do
+     ENV.update(
```

Comment:
I don't think mutating `ENV` (which has a scope that lives outside of this test case and is therefore somewhat risky to mutate from within the test case), is necessary for this test.

Can we instead just set args to the plain strings